### DISCUSSION TITLE
Elección competencia kaggle

### DISCUSSION THREAD

**Discussion by JhonierCordoba** [rramosp/ai4eng.v1#273] [00 - General]
Hola profe,

He estado mirando las competencias de kaggle y me gustaría desarrolllar esta [competencia](https://www.kaggle.com/competitions/playground-series-s3e4/overview) que trata de predecir cuando una tarjeta de crédito es fraudulenta.
Me confirmas si puedo hacer uso de esta competencia y si tienes alguna recomendación adicional.

**Comment by Felipe-RA** [comment]
Saludos @JhonierCordoba 

Tu dataset cumple las condiciones de dimensionalidad (tiene más de 30 columnas) y supera el número de filas necesario.

Ahora, te falta cumplir las los requisitos de datos faltantes y de columnas categóricas. 

Recuerda que la falta de estos dos requisitos no descarta el dataset como tal, pues puedes simularlos con distintas técnicas que se encuentran en los vídeos y las notas del curso.

¡Hasta pronto!
